### DISCUSSION TITLE
v1.7.0 - Release for Jellyfin 10.8.0

### DISCUSSION THREAD

**Discussion by iwalton3** [jellyfin/jellyfin-desktop#258] [General]
This is the first release of JMP targeting Jellyfin 10.8.0 stable. Thank you to everyone who tested the pre-releases!

Changes:
  * Update web client to 10.8.0.
  * Add --platform option. (#159)
  * Added Xbox input mapping. (#197)
  * Disable youtube-dl attempts on media errors.
  * Allow retry with transcode instead of usng a hard failure. (#127)
  * Add exit app function (#198)
  * Set the MaxStaticBitrate to avoid server hard-coded 8 mbps limit.
  * Add build for Ubuntu 22.04 (#256)
  * Enable windows dark mode support. (#247)

If you have issues with media playback due to hardware decoding (green/purple or artifacts), you can [disable it using these instructions](https://github.com/jellyfin/jellyfin-media-player/wiki/Disabling-Hardware-Decoding).

About Windows Downloads:
 - The desktop installer requires administrator rights.
 - You can avoid warnings on files by unblocking the file in the properties dialog.
 - You can manually build a release [using these instructions](https://github.com/jellyfin/jellyfin-media-player#building-for-windows).

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin-media-player/releases/tag/v1.7.0'>v1.7.0 - Release for Jellyfin 10.8.0</a>.</em>

**Comment by esmorun** [comment]
Thanks! Although... is it just me or is the Mac version broken? It fails to launch :-/

Crash report:
[MacCrash.txt](https://github.com/jellyfin/jellyfin-media-player/files/8884531/MacCrash.txt)

**Reply by iwalton3** [reply]
Yeah I'm aware of it... https://github.com/jellyfin/jellyfin-media-player/issues/259

I have a CI build here that I am awaiting a mac user to test that _might_ help: https://github.com/jellyfin/jellyfin-media-player/suites/6892375856/artifacts/267196198

**Reply by esmorun** [reply]
> Yeah I'm aware of it... #259
> 
> I have a CI build here that I am awaiting a mac user to test that _might_ help: https://github.com/jellyfin/jellyfin-media-player/suites/6892375856/artifacts/267196198

I tried it now, it works for me! 👍

**Reply by iwalton3** [reply]
Update here is the signed and fixed version: https://github.com/jellyfin/jellyfin-media-player/releases/download/v1.7.0/JellyfinMediaPlayer-1.7.0-macos.dmg

**Comment by KaXaSA** [comment]
Thanks! :P

Question: Does the SyncPlay works for you guys? 
I noticed that the 'Settings Change SyncPlay preferences' option is missing from JMP, it only appears on the web version.

![image](https://user-images.githubusercontent.com/6237603/173731741-8040a711-cd0b-4669-a9d6-fd5e5fa76f0b.png)

**EDIT:** Never mind it was some BS with the _AppData_ folders, I deleted them, and it works fine now:
```
..AppData\Local\Jellyfin Media Player
..AppData\Local\JellyfinMediaPlayer
```

(btw, why does it need 2 folders?)

**EDIT2:** So this still happens, not sure why, but I found out you can simply sign out then sign in to fix it.

**EDIT3:** [#8140](https://github.com/jellyfin/jellyfin/issues/8140)
